lib/chef-acceptance/application.rb

```diff
@@ -34,7 +53,7 @@ def run(suite, command)
             puts "Encountered an error running the recipe #{recipe}: #{e.message}\n#{e.backtrace.join("\n")}"
             if force_destroy && recipe != "destroy"
               puts "--force-destroy specified so attempting to run the destroy recipe"
```

Comment:
using `puts` in this class doesn't feel right.  Shouldn't Cli be the only class printing to console?